CHANGELOG.md

```diff
@@ -18,6 +18,14 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## 📚 Documentation -->
 
+# [0.26.3] (unreleased) - 2024-12-dd
+
+## 🐛 Fixes
+
+- **`rover persisted-queries publish` supports `clientName` for operations - @glasser #2239**
+
+  If the JSON manifest file provided to `rover persisted-queries publish` sets the `clientName` on individual operations, these client names are included in the publish instead of ignored. Additionally, the new `--for-client-name` option overrides the `clientName` field for all operations in the manifest file.
```

Comment:
The second part of this feels like a new feature instead of a fix.

```suggestion
- **`rover persisted-queries publish` publishes `clientName`s in persisted query manifests - @glasser #2239**

When using `rover persisted-queries publish`, if the JSON manifest file includes `clientName` fields on operations, those client names are now published instead of ignored.

## 🚀 Features
- **Add `--for-client-name` option to `rover persisted-queries publish` - @glasser #2239**

Use the new `--for-client-name` option to set or override the `clientName` field for all operations in a persisted query manifest.
```